nodedb.go

```diff
@@ -1036,6 +1178,24 @@ func (ndb *nodeDB) nodes() ([]*Node, error) {
 	return nodes, nil
 }
 
+func (ndb *nodeDB) legacyNodes() ([]*Node, error) {
+	nodes := []*Node{}
+
+	err := ndb.traversePrefix(legacyNodeKeyFormat.Prefix(), func(key, value []byte) error {
```

Comment:
This was the only part I had questions on. The PR to main has legacyNodeKeyFormat as a KeyFormat type instead of a FastPrefixFormatter type. Key() in FastPrefixFormatter requires a byte array input. I think Prefix() is what we should use here but would like confirmation.

Reply:
we can use both, Key() = Prefix()